### DISCUSSION TITLE
Code Server fails to finish installing on Kubernetes [air-gapped]

### DISCUSSION THREAD

**Discussion by simonjcarr** [coder/coder#10717] [General]
I am using Coder V2 on Kubernetes on a Airgap Network. I can get to the Coder UI, I can create a Template and create a Workspace. 
![image](https://github.com/coder/coder/assets/1474382/9b70285e-7970-40b8-ae8b-1f9133ad70e8)

When I click the VS Code Desktop, VS Code opens and tries to setup VS Code Server on the Workspace. it goes through the following stages and stops

* Downloading VS Code Server
* Downloading VS Code Server Locally
* Copying VS Code Server with SCP

It just hangs their and doesn't go  any further. I can login to the Workspace with the terminal, it shows that a VS Code Server install lock file has been created. If I delete the lock file and try to connect again, it just goes through the same process.

Any idea what I am doing wrong?

**Comment by matifali** [comment]
Hi @simonjcarr this issue is related https://github.com/microsoft/vscode-remote-release/issues/1242

**Comment by matifali** [comment]
Does `coder ssh <worksapce>` work?
